docs/network/interfaces_and_networks.md

```diff
@@ -563,17 +563,16 @@ Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
 ### sriov
 
-In `sriov` mode, virtual machines are directly exposed to an SR-IOV PCI
-device, usually allocated by [Intel SR-IOV device
-plugin](https://github.com/intel/sriov-network-device-plugin). The
-device is passed through into the guest operating system as a host
-device, using the
+In `sriov` mode, SR-IOV Virtual Functions' PCI devices are directly exposed to virtual machines.
+[SR-IOV device
+plugin](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin) and [CNI](https://github.com/k8snetworkplumbingwg/sriov-cni) can be used to manage SR-IOV devices in kubernetes, making them available for kubevirt to consume.
+The device is passed through into the guest operating system as a [host
+device](https://libvirt.org/drvnodedev.html), using the
 [vfio](https://www.kernel.org/doc/Documentation/vfio.txt) userspace
 interface, to maintain high networking performance.
 
 #### How to expose SR-IOV VFs to KubeVirt
-To simplify procedure, please use [SR-IOV network
-operator](https://github.com/k8snetworkplumbingwg/sriov-network-operator) to deploy
+To simplify procedure, use the [SR-IOV network operator](https://github.com/k8snetworkplumbingwg/sriov-network-operator) to deploy
 and configure SR-IOV components in your cluster. On how to use the
 operator, please refer to [their respective
 documentation](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/quickstart.md).
```

Comment:
Is this reference to the SR-IOV operator intentionally preserved?

Reply:
yes